web/index.js

```diff
@@ -141,6 +167,41 @@ function handleActions(action) {
             [];
           renderUI(currentServices, currentMessage, null, incidents);
         });
+    } else if (action.detail.name === 'getRecommendations') {
+      bdk.getBotData(roomId, botName)
```

Comment:
this should probably be broken into a separate function rather than bumping up the max-statements rule in ```.eslintrc```